test/test_issues.jl

```diff
@@ -34,4 +34,46 @@ end
         @test typeof(model_true) == Model
         @test typeof(model_true.m) == PhysicalParameter{Float32}
     end
+end
+
+@testset "Test issue #113" begin
+    @timeit TIMEROUTPUT "Issue 113" begin
+        # we are going to check 'limit_model_to_receiver_area' function for 3D with non-zero origin
+        model = example_model(n=(100,100,100), d=(10f0,10f0,10f0), o=(100f0,100f0,0f0))
+
+        # Set up 3D receiver geometry by defining one receiver vector in each x and y direction
+        nxrec = 5
+        nyrec = 3
+        xrec = range(300f0, stop=700f0, length=nxrec)
+        yrec = range(400f0, stop=600f0, length=nyrec)
+        zrec = 50f0
+
+        # Construct 3D grid from basis vectors
+        (xrec, yrec, zrec) = setup_3D_grid(xrec, yrec, zrec)
+
+        # Set up receiver structure
+        recGeometry = Geometry(xrec, yrec, zrec; dt=2, t=100f0, nsrc=1)
```

Comment:
One little critique: can you make `dt=2f0`? Same below. Everything else looks good to me!

Reply:
fixed